isso/config.py

```diff
@@ -78,10 +78,28 @@ def getboolean(self, key):
 class IssoParser(ConfigParser):
     """Parse INI-style configuration with some modifications for Isso.
 
-        * parse human-readable timedelta such as "15m" as "15 minutes"
-        * handle indented lines as "lists"
+        * Parse human-readable timedelta such as "15m" as "15 minutes"
+        * Handle indented lines as "lists"
+        * Handle quoted values
+        * Disable string interpolation ('%s') for values
     """
 
+    def __init__(self, *args, **kwargs):
+        """Supply modified defaults to configparser.ConfigParser
+        https://docs.python.org/3.9/library/configparser.html#configparser.ConfigParser
+        allow_no_value=True: Allow empty config keys
+        interpolation=None:  Do not attempt to use python string interpolation on
+                             values. This is especially important for parsing
+                             passwords that might contain '%'
+        """
+        return super(IssoParser, self).__init__(
+            allow_no_value=True, interpolation=None, *args, **kwargs)
+
+    def get(self, section, key, *args, **kwargs):
+        # Remove leading and trailing quotes from values
+        return super(IssoParser, self).get(section, key, *args, **kwargs)\
+            .strip("'").strip('"')
```

Comment:
strip can be passed multiple chars to strip off a string, no need to call it twice

Reply:
Why do we need to strip these? It seems like it would make it impossible to actually use these characters at the beginning and end of strings.

Reply:
See above.